packages/contracts/src/RelayHub.sol

```diff
@@ -89,59 +97,72 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
         setConfiguration(_config);
     }
 
+    /// @inheritdoc IRelayHub
```

Comment:
very strange. don't they inherit by default ? I suggest asking OZ to add it at least as an option, and avoid splashing all our source with this lines.
comments are useful, but not when they needlessly modify (and uglify) the source code.

(that is, require it only in case of diamonds)

Reply:
When I checked the parent natspec is not carried over by default. I don't mind it that much as it is just a oneliner comment.

Reply:
It is not a single-line, but 60-some lines, all over the code -  and a requirement to keep adding them in the future.

according to docgen creator, they are about to add it by default:
https://github.com/OpenZeppelin/solidity-docgen/issues/356#issuecomment-1032059249

So I don't see a reason to add these lines all over the place, and later remove it - especially if we're not currently publishing these docs on our doc site.